server/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java

```diff
@@ -670,14 +665,6 @@ public void testGroupByUnknownResultColumn() throws Exception {
                                 "'details_ignored['lol']' must appear in the GROUP BY clause");
     }
 
-    @Test
-    public void testGroupByUnknownGroupByColumn() throws Exception {
-        this.setup.groupBySetup();
-        Assertions.assertThrows(Exception.class,
-                                () -> execute("select max(birthdate) from characters group by details_ignored['lol']",
```

Comment:
On second thought, maybe we could keep the integration test but change it to use a cast, so we make sure to verify that it works.